tests/phpunit/tests/editor/disableBlockEditorForNavigationPostType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @group editor
+ *
+ * @covers ::_disable_block_editor_for_navigation_post_type
+ */
+class Tests_Editor_DisableBlockEditorForNavigationPostType extends WP_UnitTestCase {
+
+	const NAVIGATION_POST_TYPE     = 'wp_navigation';
+	const NON_NAVIGATION_POST_TYPE = 'wp_non_navigation';
+
+	/**
+	 * @ticket 56266
+	 *
+	 * @dataProvider data_test_it_correctly_handles_different_post_types
+	 */
```

Comment:
```suggestion
	 * @dataProvider data_test_it_correctly_handles_different_post_types
	 *
	 * @param string $post_type The post type.
	 * @param bool   $value     Whether to disable the block editor.
	 * @param bool   $expected  The expected result.
	 */
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/3005/commits/31788a26830266e44de78e6decb864d6a228e2d0